01_Core/week02/submissions/index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Cassidy's Webpage</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <div class="main">
+    <header id="home">
+        <div class="nav-bar">
+            <a href="#home" class="btn">Home</a>
+            <a href="#about" class="btn">About</a>
+            <a href="#career" class="btn">Career</a>
+            <a href="#you" class="btn">About You</a>
+            <div class="dropdown">
+                <button class="dropbtn">Social Media</button>
+                    <div class="dropdown-content">
+                        <a href="#">LinkedIn</a>
+                        <a href="#">Twitter</a>
+                        <a href="#">Facebook</a>
+                    </div>
+            </div>
+        </div>
+        <div class="sidenav">
+            <a href="www.google.com" target="_blank">Google</a>
+            <a href="www.facebook.com" target="_blank">Facebook</a>
+            <a href="www.twitter.com" target="_blank">Twitter</a>
+        </div>
+        <h1><span id="hi">Hi</span>, I'm Cassidy!</h1>
+    </header>
+    <body>
+        <div class="about">
+            <h2 id="about">About</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+        </div>
+        <div class="career">
+            <h2 id="career">Career</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+        </div>
+        <div class="form">
+            <h2 id="you">Tell me about yourself!</h2>
+        <form action="post" method="GET">
+            <div>
+                <label for="name">First Name:</label>
+                <input type="text" id="first-name">
+            </div>
+            <div>
+                <label for="name">Last Name:</label>
+                <input type="text" id="last-name">
+            </div>
+            <div>
+                <label for="gender">Gender:</label><br>
+                <input type="radio" name="gender" value="female" checked>Female<br>
```

Comment:
Using `<br>` tags here is okay, but there are better ways to handle it, particularly with CSS using `display: block`. It's an odd one, but the best process I've seen is to put have a `<label>` for each `<input>` and wrap each pair in it's own `<div>`. Then use `display: block;` on that wrapper div

Reply:
Okay, that makes sense. Thank you for your feedback!